packages/Notification/Notification.jsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
 
+import { colorWhiteLilac } from '@tds/core-colours'
 import FlexGrid from '@tds/core-flex-grid'
 import DecorativeIcon from '@tds/core-decorative-icon'
 import StandaloneIcon from '@tds/core-standalone-icon'
 import Paragraph from '@tds/core-paragraph'
 import Box from '@tds/core-box'
 import { Reveal, Fade } from '@tds/shared-animation'
+import { messaging } from '@tds/shared-styles'
 
 import safeRest from '../../shared/utils/safeRest'
-import joinClassNames from '../../shared/utils/joinClassNames'
 import { warn } from '../../shared/utils/warn'
 
-import styles from './Notification.modules.scss'
+const NotificationContainer = styled(({ variant, ...rest }) => <Box {...rest} />)(
```

Comment:
Let's prepend the names for styled components with `Styled` as a naming convention.